### PR TITLE
Improve image-compare widget

### PR DESCRIPTION
**Pull request by ickk**
- add UI handle & cursor affordances to visually indicate interactivity to end-user,
- add padding fix to image-titles,
- add slight blur-shadow to image-titles to improve readability,
- allow widget-user to override slider configuration.

generally improves code quality

before & after: 
image

### CODE REVIEW

File: content/news/2023-07-09-bevy-0.11/index.md [hunk 1]
```diff
@@ -291,7 +291,7 @@ differently, we get those awkward specular highlights. With parallax mapping,
 they are gone.
 
 <b style="display:block; margin-bottom: -18px">Drag this image to compare</b>
-<div class="image-compare" style="aspect-ratio: 16 / 9" data-title-a="Normals Only" data-title-b="Parallax+Normals">
+<div class="image-compare" style="aspect-ratio: 16 / 9; --slider-value: 39%" data-title-a="Normals Only" data-title-b="Parallax+Normals">
```
Comment (ickk) [inline content/news/2023-07-09-bevy-0.11/index.md:294]: This just improves the utility of this image when js is disabled I think
before (js disabled):
image
after (js disabled):
image

**Review by IceSentry — APPROVED** [review]
The min and max value don't seem to work for me. I'm using the vivaldi browser, so chromium.

I don't really mind this though. I prefer not having the min/max personally.

**Comment by ickk** [comment]
> The min and max value don't seem to work for me. I'm using the vivaldi browser, so chromium.

It's really weird that Vivaldi differs from Chrome, Edge, Ungoogled-chromium 🤔

**Comment by IceSentry** [comment]
Yeah, not sure what's going on. I can confirm it works in edge and chrome though, so I say ship it.

**Comment by ickk** [comment]
Looking into it a little bit, it appears that Vivaldi doesn't do anything with the `<input type="range">`'s `min`/`max` values. This is probably a bug in the browser (I couldn't find a caniuse listing, or a bug report or anything for Vivaldi).

**Review by cart — APPROVED** [review]

**Pull request merged**